services/src/main/java/org/exoplatform/addons/gamification/service/setting/zone/impl/ZoneRegistryImpl.java

```diff
@@ -68,8 +60,8 @@ public void start() {
       try {
         // Processing registered domains
         for (ZoneConfig domain : zoneMap.values()) {
-          DomainDTO domainDTO = domainService.findDomainByTitle(domain.getZoneName());
-          if (domainDTO == null) {
+          DomainDTO domainDTO = domainService.getDomainByTitle(domain.getZoneName());
+          if (domainDTO == null || domainDTO.isDeleted()) {
```

Comment:
If a domain is deleted, will it be recreated in next restart ?

Reply:
it is expected that the automatically created domains should be recreated if not found

Reply:
@GouadriaHanen it will be better to forbid deleting them instead, WDYT? cc @mzorai

Reply:
there is too much behavior that should be forbidden (delete and rename) but as discussed with @mzorai  in the next iteration there will be too many changes in the behavior of the domains so we decided to just fix the bug in the achievement table and the double creating of disabled domain